quickstarts/java/elasticsearch-query/config.yml

```diff
@@ -0,0 +1,32 @@
+name: elasticsearch-query
+title: Elasticsearch query Java Agent Extension
+description: |
+  # What is elasticsearch?
+  Elasticsearch is a search engine based on the Lucene library. It provides a distributed, multitenant-capable full-text search engine with an HTTP web interface and schema-free JSON documents.
```

Comment:
You may want to specify in the description this is specific for java.

Reply:
Added reference to java and java agent , i fixed logo, it is rendering good in https://www.svgviewer.dev/ .